Anomaly/Patches/AbilityDefs/Abilities_Anomaly.xml

```diff
@@ -62,8 +62,8 @@
 		<value>
 			<projectile Class="CombatExtended.ProjectilePropertiesCE">
 				<damageAmountBase>8</damageAmountBase>
-				<armorPenetrationSharp>3.85</armorPenetrationSharp>
-				<armorPenetrationBlunt>4.22</armorPenetrationBlunt>
+				<armorPenetrationSharp>9</armorPenetrationSharp>
```

Comment:
Why does the fingerspike spike have higher AP than the gorehulk spike?

Reply:
They're not actually that small, the size of a dog is not small for something of eldritch origins that shoots spikes with 100% sharp AP.